plugins/modules/ndo_vlan_pool.py

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2024, Akini Ross (@akinross) <akinross@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: ndo_vlan_pool
+short_description: Manage VLAN Pools on Cisco Nexus Dashboard Orchestrator (NDO).
+description:
+- Manage VLAN Pools on Cisco Nexus Dashboard Orchestrator (NDO).
+author:
+- Akini Ross (@akinross)
+options:
+  template:
+    description:
+    - The name of the template.
+    - The template must be a fabric policy template.
+    type: str
+    required: true
+  vlan_pool:
+    description:
+    - The name of the VLAN Pool.
+    type: str
+    aliases: [ name ]
+  vlan_pool_uuid:
+    description:
+    - The uuid of the VLAN Pool.
+    - This parameter is required when the O(vlan_pool) needs to be updated.
+    type: str
+    aliases: [ uuid ]
+  description:
+    description:
+    - The description of the VLAN Pool.
+    type: str
+  vlan_ranges:
+    description:
+    - A list of vlan ranges attached to the VLAN Pool.
+    - The list of configured vlan ranges must contain at least one entry.
+    - When the list of vlan ranges is null the update will not change existing entry configuration.
+    type: list
+    elements: dict
+    suboptions:
+      from_vlan:
```

Comment:
The examples say `from` and `to` not `from_vlan` `to_vlan`
I think `from` and `to` is good because the dictionary name already specifies `vlan_ranges`. 
Unless an alias can be used.

Reply:
added to aliases